src/helpers/LanguageHelper.js

```diff
@@ -0,0 +1,23 @@
+class LanguageHelper {
+  static determineExtension(language) {
```

Comment:
I would prob do this instead:

```suggestion
 static determineExtension(language) {
    if (language === 'lua') {
        return '.lua';
    }
    if (language === 'C#') {
        return '.cs';
    }
    return '.js';
}
```

Note that the extension for C# is '.cs' not '.c'.